dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/MethodInvocationNode.java

```diff
@@ -76,6 +96,28 @@ public TreePath getTreePath() {
         return treePath;
     }
 
+    /**
+     * If this MethodInvocationNode is a node for an {@link Iterator#next()} desugared from an
+     * enhanced for loop, then return the expression in the for loop, e.g., {@code iter} in {@code
+     * for(Object o: iter}. Otherwise, return null.
+     *
+     * @return the iter expression, or null if this is not a {@link Iterator#next()} from an
+     *     enhanced for loop
+     */
+    public @Nullable ExpressionTree getIteratorExpression() {
+        return iteratorExpression;
+    }
+
+    /**
+     * Set the iter expression.
```

Comment:
Can you make "iter expression" more descriptive?  Even changing it to a term that is used elsewhere in the code would be better, I think.